src/main/java/net/sf/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -119,6 +124,16 @@ public static ParserResult parse(Reader in, ImportFormatPreferences importFormat
         return Optional.of(entries.iterator().next());
     }
 
+    @Override
+    public List<BibEntry> parseEntries(InputStream inputStream) throws ParserException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8));
```

Comment:
This is just to parse a bunch of BibEntries which may come from anywhere. Importing / opening a database uses the other parse method which uses the proper encoding specified in the file or preferences.